queries/ne-places.jinja2

```diff
@@ -0,0 +1,41 @@
+{% extends 'ne.jinja2' %}
+{% block query %}
+SELECT
+
+  gid AS __id__,
+
+  {% filter geometry %}the_geom{% endfilter %} AS __geometry__,
+  NULL::bytea AS __label__,
+
+  {{ ne_common_properties() }},
+
+  NULL::jsonb AS __boundaries_properties__,
```

Comment:
Just checking - we still need these `NULL::jsonb AS __layer_properties__` to make the columns match across the `UNION ALL`s?